scripts/grep_pv

```diff
@@ -16,21 +16,24 @@ fi
 KEYWORD=$1
 
 if [[ -n $KEYWORD ]]; then
+    # shellcheck disable=SC2164
     cd /reg/d/iocData/
```

Comment:
Thinking more critically about the code path here, maybe we can remove this `cd` command entirely instead of disabling shellcheck. This is always overridden by whatever is going on in the loop.